docs/user/03-11-customize_secret.md

```diff
@@ -0,0 +1,62 @@
+# Customize the Default Credentials and Access
+
+> [!NOTE]
+> This feature is still being developed and will be available soon.
+
+You can customize the `sap-btp-manager` Secret and manage your own default configuration of the SAP BTP Operator module.
+
+## Context
+
+When you create SAP BTP, Kyma runtime, the `sap-btp-manager` Secret is automatically created as the default Secret managing the SAP BTP Operator's resources. 
+Because of Kyma's automatic reconciliation, any changes to the Secret are reverted, and the previous settings are restored within up to 24 hours.
```

Comment:
```suggestion
Because of Kyma's automatic reconciliation, any changes to the Secret are reverted, and the previous settings are restored within 24 hours.
```
Do we need this?

Reply:
Not necessarily. Removed.